sumologic/util.go

```diff
@@ -294,3 +301,96 @@ func GetTimeRangeBoundary(tfRangeBoundary map[string]interface{}) interface{} {
 
 	return nil
 }
+
+func SuppressEquivalentTimeDiff(isRelative bool) func(k, oldValue, newValue string, d *schema.ResourceData) bool {
+	return func(k, oldValue, newValue string, d *schema.ResourceData) bool {
+		var handleError = func(err error) bool {
+			log.Printf("[WARN] Time value could not be converted to seconds. Details: %s", err.Error())
+			return false
+		}
+
+		if oldValue == "" || newValue == "" {
+			return false
+		}
+
+		if oldValInSeconds, oldErr := getTimeInSeconds(oldValue); oldErr != nil {
```

Comment:
Is this the correct "go" way of handling errors?

Reply:
yeah, this looks right to me.